instrument.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
 # guess OS_TYPE if not provided
+OS_TYPE=${OS_TYPE:-}
```

Comment:
Why this change?

Reply:
Ops, this was to be part of #2129 to keep the code consistent between the two scripts. Without that the script fails on `if [ -z "$OS_TYPE" ]; then`: it doesn't like that test without defining `OS_TYPE` first.